.issues/20260730_기능개선_배터리_제한없음_없이_백그라운드_유지.md

```
@@ -0,0 +1,26 @@
+# 🚀 [기능개선][Android] 배터리 제한없음 설정 없이 백그라운드 게임 유지
```

markdownlint-cli2 (0.23.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "android"

(MD052, reference-links-images)